letsencrypt-win-simple/Plugin/OverviewPlugin.cs

```diff
@@ -0,0 +1,154 @@
+
+        public override void PrintMenu()
+        {
+            Console.WriteLine(" O: Get overview of all bindings, protocols, renewals and certificates");
+        }
+
+        public override void HandleMenuResponse(string response, List<Target> targets)
+        {
+            if (response == "o")
+            {
+                Console.WriteLine("Running Overview Plugin");
+                Log.Information("Running Overview Plugin");
+
+                GetIisSites();
+                Console.WriteLine();
+                Console.WriteLine("Press enter to continue.");
+                Console.ReadKey();
+                GetRenewals();
+                Console.WriteLine();
+                Console.WriteLine("Press enter to continue.");
+                Console.ReadKey();
+                GetCertificates();
+                Console.WriteLine();
+            }
+        }
+
+        public static void GetIisSites()
+        {
+            using (var iisManager = new ServerManager())
+            {
+                Console.WriteLine("IIS Sites:");
+                foreach (var site in iisManager.Sites)
+                {
+                    Console.WriteLine("  " + site.Id + ": " + site.Name);
+                    foreach (var binding in site.Bindings.OrderBy(o => o.Host))
+                    {
+                        if (!String.IsNullOrEmpty(binding.Host))
+                        {
+                            var line = "     - " + binding.Host + ((binding.Protocol == "https") ? " (https)" : "");
+                            Console.WriteLine(line);
+                        }
+                    }
+                }
+            }
+        }
+
+        public static void GetCertificates()
+        {
+            try
+            {
+                Console.WriteLine();
+                Console.WriteLine("Certificates:");
+                string _certificateStore = Properties.Settings.Default.CertificateStore;
+                var store = new X509Store(_certificateStore, StoreLocation.LocalMachine);
+                store.Open(OpenFlags.OpenExistingOnly | OpenFlags.ReadWrite);
+                var certificates = store.Certificates;
+                var i = 0;
+                foreach (var certificate in certificates)
+                {
+                    i++;
+                    Console.WriteLine("  " + certificate.FriendlyName);
+                }
+                Console.WriteLine();
+                Console.WriteLine("  " + i + " certificates in total");
+                store.Close();
+            }
+            catch (Exception ex)
+            {
+                Log.Error("Error encountered while opening certificate store. Error: {@ex}", ex);
+                throw new Exception(ex.Message);
+            }
+        }
+
+        public static void GetRenewals()
+        {
+            try
+            {
+                var _settings = new Settings(ClientName, Program.Options.BaseUri);
```

Comment:
This main code for getting renewals shouldn't be in your plugin but in the main program.

Then you should run it to get all of the Renewals and can then print them out under your plugin.